app/Http/Resources/V1/SearchResource.php

```diff
@@ -18,9 +18,11 @@ public function toArray($request)
     {
         return [
             'id' => $this->id,
+            'playlist_id' => $this->playlist_id,
```

Comment:
We are using the same "SearchResource.php" for Project, Playlist and Activity.
Since "playlist_id" attribute will not exist for Project and Playlist,
we will need to add check similar to this one for including "playlist_id" in response when present.

`'playlist_id' => $this->when(isset($this->playlist_id), $this->playlist_id),`